project/build.sbt

```diff
@@ -33,6 +35,7 @@ unmanagedSourceDirectories in Compile ++= {
     root / "tools/shared/src/main/scala",
     root / "tools/jvm/src/main/scala",
     root / "js-envs/src/main/scala",
+    root / "js-envs-test-kit/src/main/scala",
```

Comment:
This shouldn't be needed (and therefore, nor the dependency on `junit` on line 20)